en/intro/conventions.rst

```diff
@@ -7,10 +7,19 @@ conventions, you get free functionality, and you liberate yourself from the
 maintenance nightmare of tracking config files. Conventions also make for a very
 uniform development experience, allowing other developers to jump in and help.
 
+.. note::
+
+    CamelCase is the practice of writing compound words or phrases so that each word or abbreviation in the middle of the sentence begins with an uppercase letter.
+
+    PascalCase is the practice of writing compound words or phrases so that each word or abbreviation begins with an uppercase letter.
```

Comment:
```suggestion
    Pascal case is the practice of writing compound words or phrases so that each word including the first begins with an uppercase letter. E.g. ``RedApple``.
```